storage/adls_gen2_mount.go

```diff
@@ -19,10 +21,23 @@ type AzureADLSGen2Mount struct {
 	InitializeFileSystem bool   `json:"initialize_file_system"`
 }
 
+func getAzureDomain(client *common.DatabricksClient) string {
+	domains := map[string]string{
+		"PUBLIC":       "core.windows.net",
+		"USGOVERNMENT": "core.usgovcloudapi.net",
+		"CHINA":        "core.chinacloudapi.cn",
+	}
```

Comment:
Is this list published somewhere?

Reply:
yes - you can get it with azure-cli: https://github.com/databricks/terraform-provider-databricks/issues/4272#issuecomment-2504650693 and in the corresponding docs: https://learn.microsoft.com/en-us/azure/reliability/sovereign-cloud-china